core/templates/dev/head/pages/exploration-player-page/suggestion-modal-for-learner-local-view/suggestion-modal-for-exploration-player.service.ts

```diff
@@ -61,7 +61,14 @@ angular.module('oppia').factory('SuggestionModalForExplorationPlayerService', [
             $timeout(function() {
               $scope.showEditor = true;
             }, 500);
-
+            $scope.detectChange = function(e) {
+              if ($($scope.suggestionData.suggestionHtml).html() ===
+              ( $($scope.originalHtml).html())) {
+                return true;
+              } else {
```

Comment:
You don't need this else block, since the if block has a return. Just have return false at the end.